torch/onnx/_internal/fx/fx_onnx_interpreter.py

```diff
@@ -248,6 +248,11 @@ def _fill_tensor_shape_type(
                 expected_value
             )
             onnxscript_value.shape = torch.Size([1])
+        elif isinstance(expected_value, (int, float, bool)):
```

Comment:
Out of curiosity, do we care about `complex`?

Reply:
I think no, because there is no SymComplex